vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -32,6 +33,7 @@ def self.recent
 
     def mark_orphaned!
       update!(orphaned_at: Time.current)
+      order_detail.update_order_status!(user, OrderStatus.complete.first) if order_detail.present?
```

Comment:
Can we use `OrderStatus.complete_status`? I'd like for us to eventually get to the point of being able to use `OrderStatus.complete` as the reference to the actual status, but that's a tedious change.